hack/make-aggregator-pki.sh

```diff
@@ -7,6 +7,21 @@ set -o pipefail
 rm -rf data/grafana-aggregator
 
 mkdir -p data/grafana-aggregator
+
 openssl req -nodes -new -x509 -keyout data/grafana-aggregator/ca.key -out data/grafana-aggregator/ca.crt
-openssl req -out data/grafana-aggregator/client.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/client.key -subj "/CN=development/O=system:masters"
-openssl x509 -req -days 365 -in data/grafana-aggregator/client.csr -CA data/grafana-aggregator/ca.crt -CAkey data/grafana-aggregator/ca.key -set_serial 01 -sha256 -out data/grafana-aggregator/client.crt
+openssl req -out data/grafana-aggregator/client.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/client.key \
+  -subj "/CN=development/O=system:masters" \
+  -addext "extendedKeyUsage = clientAuth"
+openssl x509 -req -days 365 -in data/grafana-aggregator/client.csr -CA data/grafana-aggregator/ca.crt -CAkey data/grafana-aggregator/ca.key \
+  -set_serial 01 \
+  -sha256 -out data/grafana-aggregator/client.crt \
+  -copy_extensions=copyall
+
+openssl req -out data/grafana-aggregator/server.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/server.key \
+  -subj "/CN=localhost/O=aggregated" \
+  -addext "subjectAltName = DNS:v0alpha1.example.grafana.app.default.svc,DNS:localhost" \
```

Comment:
When ExternalName is using a value of `localhost` for `Host`, I am noticing a further validation against SAN of `v0alpha1.example.grafana.app.default.svc` where `name` of `ExternalName` is `v0alpha1.example.grafana.app`. As such, added this extra SAN value.